### ISSUE TITLE
PyTorch nightly version required?

### ISSUE THREAD

**Issue by cipher009** [ElementAI/wise_ils#4]
In Requirements:
Pytorch version 0.4 or higher.

However, the maskrcnn_benchmark used in models/mrcnn.py requires PyTorch from a nightly release.
Am I missing something?

**Comment by IssamLaradji** [comment]
Yes, maskrcnn_benchmark got updated quickly that some dependencies got broken. I will adapt the code to use the trochvision maskrcnn_benchmark. Thanks for pointing this out.

**Comment by d12306** [comment]
Hi, @IssamLaradji , thanks for your implementation, just to make sure I understand corrently, so the prm_point_dict.json is obtained after training using PRM,  and proposal_dict.json is the coarse segmentation proposals obtain by MCG, right?
if I want to train on a new dataset, such as COCO dataset, I first need to get the MCG proposals and then train the PRM to obtain the peak points, right?

**Comment by Haitham96** [comment]
Hi all, 
i installed **pytorch** through anaconda by this commands:
```
conda install -c pytorch torchvision-nightly
conda install -c pytorch torchvision
```
unfortunately, **maskrcnn_benchmark** dependencies got **broken**
```
       from maskrcnn_benchmark.structures.bounding_box import BoxList
ModuleNotFoundError: No module named 'maskrcnn_benchmark'
```
How can i fix it?
